closed/test/jdk/openj9/internal/security/property-java.security

```diff
@@ -50,7 +50,7 @@ RestrictedSecurity.TestBase.Version-Extended.tls.disabledAlgorithms =
 RestrictedSecurity.TestBase.Version-Extended.jce.provider.1 = sun.security.provider.Sun
 RestrictedSecurity.TestBase.Version-Extended.jce.provider.2 = sun.security.rsa.SunRsaSign
 RestrictedSecurity.TestBase.Version-Extended.jce.provider.3 = sun.security.ec.SunEC
-RestrictedSecurity.TestBase.Version-Extended.jce.provider.4 = sun.security.ssl.SunJSSE
+RestrictedSecurity.TestBase.Version-Extended.jce.provider.4 = com.sun.net.ssl.internal.ssl.Provider
```

Comment:
Again, shouldn't `RestrictedSecurity.Test-Profile.Base.desc.hash` also need to change?

Reply:
Same as above, in the `TestProperties.java` file, all profiles except for `Test-Profile-SameStartWithoutVersion` are designed to test error messages. In these profiles, the error message is returned before the hash value is checked, so the hash value is not used.